libs/island-ui/core/src/lib/Toast/Toast.css.ts

```diff
@@ -14,7 +14,7 @@ globalStyle(`${root} .Toastify__toast-container`, {
   transform: 'translate3d(0, 0, 9999px)',
   position: 'fixed',
   padding: '4px',
-  width: '320px',
+  width: '432px',
```

Comment:
Do we need to think about screens smaller than this? F.x. iPhone 12 Pro is 390px

Reply:
Ah nvm didn't see the media query below